cli/wadl2checker/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Checker.scala

```diff
@@ -181,24 +78,21 @@ object Wadl2Checker {
         }
       }
 
-      Some((source, result, metaOutResult, c))
-    } else {
-      // arguments are bad, usage message will have been displayed
-      None
+      Some((source, result, metaOutResult, config))
```

Comment:
This would be the place that is maybe helped out by using the mutable parsing. You get an `Option` return, so you can just `flatMap` it rather than having the conditional here. That's not really much of a change, though.